lxd/device/gpu_physical.go

```diff
@@ -654,21 +654,21 @@ func (d *gpuPhysical) postStop() error {
 
 // deviceNumStringToUint32 converts a device number string (major:minor) into separare major and
 // minor uint32s.
-func (d *gpuPhysical) deviceNumStringToUint32(devNum string) (uint32, uint32, error) {
+func (d *gpuPhysical) deviceNumStringToUint32(devNum string) (major uint32, minor uint32, err error) {
```

Comment:
should be a separate linter commit to allow backporting

Reply:
fixed.